src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -132,9 +132,17 @@ Real64 EIRPlantLoopHeatPump::getLoadSideOutletSetPointTemp(EnergyPlusData &state
     } else if (thisLoadPlantLoop.LoopDemandCalcScheme == DataPlant::LoopDemandCalcScheme::DualSetPointDeadBand) {
         if (thisLoadComp.CurOpSchemeType == DataPlant::OpScheme::CompSetPtBased) {
             // there will be a valid set-point on outlet
-            return state.dataLoopNodes->Node(this->loadSideNodes.outlet).TempSetPointHi;
+            if (this->EIRHPType == DataPlant::PlantEquipmentType::HeatPumpEIRCooling) {
+                return state.dataLoopNodes->Node(this->loadSideNodes.outlet).TempSetPointHi;
+            } else {
+                return state.dataLoopNodes->Node(this->loadSideNodes.outlet).TempSetPointLo;
+            }
```

Comment:
`HeatPump:PlantLoop:EIR:Cooling` use the maximum temperature setpoint and `HeatPump:PlantLoop:EIR:Heating` use the minimum temperature setpoint.

Reply:
Got it, and this works for both loop and component setpoint control.  Seems like a great fix to me.